tests/suites/test_suite_ecp.function

```diff
@@ -988,6 +1014,41 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_ECP_C */
```

Comment:
If I understand https://jenkins-mbedtls.oss.arm.com/job/mbed-tls-pr-head/job/PR-7815-head/3/execution/node/7001/log/ correctly, outcome analysis complains that the `ecp_calc_public` tests are not executed when all EC algorithms use PSA. But I'd expect that, since the test function depends on `MBEDTLS_ECP_C`. Am I misunderstanding the test report? What did I do wrong?

Reply:
The way parity testing is currently done, you need to add the description of the tests that are expected to be skipped to [https://github.com/Mbed-TLS/mbedtls/blob/development/tests/scripts/analyze_outcomes.py#L213](this list) in `analyze_outcomes.py`.

Reply:
Wow, that's pretty clumsy. I see there's a mechanism to ignore whole test suites, and individual test cases, but not test functions. Are you going to add one? Or should we split test suites into our-implementation-only stuff vs also-with-driver stuff?

Reply:
In the meantime we've added regex support to exclude a family of test cases with similar names. I don't think ignoring based on test function would be doable without some big changes, as the analysis is based on the outcomes files doesn't record the test function unless I'm mistaken.

If the list becomes too clumsy, splitting the test suite is indeed a good idea. In this case I don't think the split would be our-implementation-only vs also-with-drivers: it would rather be `ECP_LIGHT` vs "the rest of `ECP_C`".